Sources/StreamChatUI/Resources/en.lproj/Localizable.strings

```diff
@@ -262,5 +275,13 @@
 
 /// Alert title when closing a poll.
 "alert.poll.end-title" = "Nobody will be able to vote in this poll anymore.";
+/// Alert title when adding a comment to a poll.
+"alert.poll.add-comment" = "Add a Comment";
+/// Alert title when adding a new poll suggestion.
+"alert.poll.suggest-option" = "Suggest an Option";
+/// Alert title when updating a comment to a poll.
+"alert.poll.update-comment" ="Update your Comment";
```

Comment:
Shouldn't alert titles use the sentence-style capitalization ([link](https://developer.apple.com/design/human-interface-guidelines/alerts))?

Reply:
I followed Figma design, but if there is a guideline from Apple, we should do it. I'll update this 👍

Reply:
Done 👍